modules/social_features/social_poll/social_poll.info.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Social Poll'
+description: 'Social Poll'
+type: module
+core: 8.x
+dependencies:
+  - drupal:poll
+  - drupal:block
+  - drupal:field
+  - social:social_landing_page
```

Comment:
i don't think we want this hard dependency, poll should be able to work without social_landing_pages.